src/components/Spawn/ExpirationField.tsx

```diff
@@ -103,7 +103,7 @@ export const ExpirationField: React.VFC<ExpirationFieldProps> = ({
           >
             <span>
               <PaddedCheckbox
-                data-cy="neverExpireCheckbox"
+                data-cy="never-expire-checkbox"
                 disabled={disableExpirationCheckbox}
                 label="Never"
                 checked={noExpiration}
```

Comment:
This doesn't check automatically when ever i try to spawn a virtual workstation like it currently does.

Reply:
Thanks!